src/eth/EthereumTokenService.js

```diff
@@ -75,6 +75,10 @@ export default class EthereumTokenService extends PrivateService {
       return new PethToken(contract, this.get('web3'), tub);
     }
 
+    if (symbol === tokens.COL5) {
```

Comment:
I don't think tokens.COL5 is defined

Reply:
We can’t hardcode this here—it should rely on the `decimals` value from the config

Reply:
Removed this and added it to the tokens config in dai-plugin-config https://github.com/makerdao/dai-plugin-config/blob/master/src/utils/configHelper.js#L52